en/docs/assets/css/devanttheme.css

```
@@ -763,5 +763,10 @@ html .md-nav--secondary .md-nav__link:not(.md-nav__link--passed) {
 }
 
 @media screen and (max-width: 59.9375em) {
+    .md-content {
+        padding-left: 0.6rem;
+        padding-right: 0.6rem;
+    }
+
     body[data-md-scrolllock] {}
```

Stylelint (17.3.0)

[error] Unexpected empty block (block-no-empty)

(block-no-empty)